model/Security/Vocabularies/CvssSeverityType.md

```diff
@@ -8,7 +8,23 @@ Specifies the CVSS base, temporal, threat, or environmental severity type.
 
 ## Description
 
-CvssSeverityType specifies the CVSS severity type, defined in the CVSS specifications as the textual representation of the numeric CVSS score. The severity type entries are inclusive of and applicable to enumerations found in CVSS versions [3](https://www.first.org/cvss/v3.0/specification-document#Qualitative-Severity-Rating-Scale) and [4](https://www.first.org/cvss/v4.0/specification-document#Qualitative-Severity-Rating-Scale). CvssSeverityType is a mandatory field because baseSeverity is required in the CVSS version [3.0](https://www.first.org/cvss/cvss-v3.0.json), [3.1](https://www.first.org/cvss/cvss-v3.1.json), and [4.0](https://www.first.org/cvss/cvss-v4.0.json) schemas. The field can be used to document the base, temporal, threat, or environmental severity.
+CvssSeverityType specifies the Common Vulnerability Scoring System (CVSS)
+severity type, defined in the CVSS specifications as the textual representation
+of the numeric CVSS score.
+
+The severity type entries are inclusive of and applicable to enumerations found
+in
+[Common Vulnerability Scoring System v3.0: Specification Document](https://www.first.org/cvss/v3.0/specification-document#Qualitative-Severity-Rating-Scale)
+and
+[Common Vulnerability Scoring System version 4.0: Specification Document](https://www.first.org/cvss/v4.0/specification-document#Qualitative-Severity-Rating-Scale).
```

Comment:
Why are the links on separate lines and `in` and `and` have their own line? This seems like odd formatting.

Reply:
Please see a comment below for CvssSeverityType.md. Thank you.